expenses-manager-backend/src/main/java/com/endava/expensesmanager/service/impl/ExpenseServiceImpl.java

```diff
@@ -109,4 +109,8 @@ public List<ExpenseDto> getExpensesPage(int userId, LocalDateTime startDate, Loc
                 .toList();
     }
 
+    public Integer countExpensesPage(int userId, LocalDateTime startDate, LocalDateTime endDate, int size, Integer categoryId, Integer currencyId) {
+        int expensesCount=expenseRepository.countAllExpenses(userId, startDate, endDate, categoryId, currencyId);
+        return (expensesCount-1)/size+1;
```

Comment:
aici trebuie sa ne asiguram ca in cazurile in care avem rezultatul 1.1 pagini sa returnam 2. Vezi daca Math.ceil() face ce trebuie

Reply:
Din moment ce e integer, nu calculează deja câtul fără virgulă?

Reply:
daca ai 10 elemente pe care le imparti in pagini de cate 6 pe logica actuala tu faci 9/7 care e 1.28, care convertit la int este 1, lucru care nu este ok pentru ca e nevoie de 2 pagini, prima va avea 6 elemente si a 2a va avea 4 elemente. De asta am spus ca poti sa incerci sa folosesti Math.ceil() ca sa rotunjesti superior